src/pages/Editor/components/ShapeInfo.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const ShapeInfo = ({ shape }) => {
+    const GRID_SIZE = 20;
+    const canvasHeight = 600;
+    if (!shape) {
+        return <div>Select a shape to see its details.</div>;
+    }
+
+    const convertCoordinatesForDisplay = (point) => {
```

Comment:
I think, that here you can use new function creation syntax like 
function convertCoordinatesForDisplay (point){
    //your code
}